package.json

```diff
@@ -16,7 +16,7 @@
     "@typescript-eslint/eslint-plugin": "^2.34.0",
     "@typescript-eslint/parser": "^2.34.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-eslint-plugin": "^2.3.0",
+    "eslint-plugin-eslint-plugin": "^3.6.1",
```

Comment:
Is it possible to use version 4? https://github.com/eslint-community/eslint-plugin-eslint-plugin/blob/v4.4.1/package.json#L71-L73
Or is it possible to change the ESLint version that the plugin supports and use version 5?

Reply:
I think the supported Node version should be in line with what ESLint supports

Reply:
`eslint-plugin-eslint-plugin` will be included when going to ESLint v8

I think the supported Node version should be in line with what ESLint supports

Reply:
`eslint-plugin-eslint-plugin` will be included when going to ESLint v8

I also think the supported Node version should be in line with what ESLint supports

Reply:
I believe `eslint-plugin-eslint-plugin` v4 supports ESLint v6. So I think we can use v4.
I don't think we should wait for ESLint v8.
https://github.com/eslint-community/eslint-plugin-eslint-plugin/blob/v4.4.1/package.json#L71-L73